test/system/zsystemTest_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020  InterDigital Communications, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
```

Comment:
it is used to destroy the sandbox only that got created as part of the test cases.... not sure how to deal with it otherwise since there are no uninit() function... only an init()...

Reply:
Leave it as is for now and we can address as a feature improvement later.